integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonMergerRDD.scala

```diff
@@ -495,7 +628,22 @@ class CarbonMergerRDD[K, V](
     }
   }
 
-
+  private def createOrderingForColumn(column: CarbonColumn): Ordering[Object] = {
```

Comment:
Same method is present org.apache.carbondata.spark.load.DataLoadProcessBuilderOnSpark#createOrderingForColumn , please move this to some utility

Reply:
Removed and used the function in org.apache.carbondata.spark.load.DataLoadProcessBuilderOnSpark only.